src/main/java/teammates/ui/controller/StudentProfileEditSaveAction.java

```diff
@@ -9,12 +10,12 @@
 import teammates.common.util.StatusMessage;
 import teammates.common.util.StatusMessageColor;
 import teammates.common.util.StringHelper;
-
 /**
  * Action: saves the new profile details given by a student.
  *         A purely Action based URI as it redirects back to
  *         StudentProfilePageAction once completed
  */
+
```

Comment:
Can remove these whitespace-only changes?

Reply:
Sir (@wkurniawan07 ) Sorry for being so late This white space Can't be removed as without it checkstyle test do not pass . there should be gap between comment and class

Reply:
??? that cannot be the case; if it is, the code in its previous state would have been flagged by Checkstyle.

Reply:
@wkurniawan07 sir what should I do in that case

Reply:
Revert it to how it was before.